src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -24,6 +24,12 @@ You can configure the `ZipkinExporter` through
    associated with the telemetry has "service.name" defined, then it'll be
    preferred over this option.
 * `Endpoint`: URI address to receive telemetry (default `http://localhost:9411/api/v2/spans`).
+
+   It can be configured using two ways:
+   1. [Incode](../../examples/Console/TestZipkinExporter.cs)
```

Comment:
"Incode" isn't a word I don't think. How about "Via code" and "Via environment variable" instead?

Reply:
@CodeBlanch - I fixed it now. Please review. Thanks!

Reply:
Looks good!